pkg/features/kube_features.go

```diff
@@ -1167,7 +1168,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	StatefulSetAutoDeletePVC: {Default: true, PreRelease: featuregate.Beta},
 
-	StatefulSetStartOrdinal: {Default: true, PreRelease: featuregate.Beta},
+	StatefulSetStartOrdinal: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.31, remove in 1.32
```

Comment:
Nit, remove in 1.33, we usually do GA+2 for removal.

Reply:
Updated to 1.33 for removal.